docs/release-notes/november-2024.md

```diff
@@ -0,0 +1,121 @@
+---
+slug: november-2024
+hide_title: true
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+title: November 2024
+tags:
+  - Release notes
+---
+# Semgrep release notes for November 2024
+
+## 🌐 Semgrep AppSec Platform
+
+### Added
+
+### Changed
+
+### Fixed
+
+## 💻 Semgrep Code
+
+### Added
+
+- Added the ability to filter Semgrep Code findings by **Last fixed** and **Last triaged** dates in Semgrep AppSec Platform.
```

Comment:
I will have to move this line on **Last fixed** and **Last triaged* to Semgrep AppSec Platform because this feature is live for all products, not just Code.